File: src/main/java/com/kingdom/repository/GameHistoryRepository.java
package com.kingdom.repository;

import com.kingdom.model.GameHistory;
import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface GameHistoryRepository extends CrudRepository<GameHistory, Integer> {

    List<GameHistory> findTop80ByOrderByGameIdDesc();
}

File: src/main/kotlin/com/kingdom/repository/GameHistoryRepository.kt
package com.kingdom.repository

import com.kingdom.model.GameHistory
import org.springframework.data.jpa.repository.Query
import org.springframework.data.repository.CrudRepository

interface GameHistoryRepository : CrudRepository<GameHistory, Int> {

    fun findTop80ByOrderByGameIdDesc(): List<GameHistory>

}
